packages/components/src/local/adjudicate-turn-form/__snapshots__/index.spec.tsx.snap

```diff
@@ -121,7 +121,7 @@ exports[`AdjudicateTurnForm renders correctly 1`] = `
         className="container main-container align-right"
       >
         <section
-          className="input-container"
+          className="input-container input-container"
```

Comment:
@lebaphi - do you know why the class name is being provided twice?

Reply:
@IanMayo I'm not sure, it's auto generated by jest 🤔

Reply:
Sorry, of course it is. Let's not worry about it.